manual/googlechrome-extensions/googlechrome-emberinspector/googlechrome-ember-inspector.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-ember-inspector</id>
+    <version>1.0.0</version>
+    <title>Ember Inspector</title>
+    <authors>Ember Development Team</authors>
+    <owners>gep13 chocolatey</owners>
+    <projectUrl>http://emberjs.com/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/d24c0fffa284ccddcb6d8ad862dd77c7508b97df/icons/googlechrome-ember-inspector.png</iconUrl>
+    <projectSourceUrl>https://github.com/emberjs/ember-inspector</projectSourceUrl>
+    <docsUrl>https://github.com/emberjs/ember-inspector/blob/master/README.md</docsUrl>
+    <bugTrackerUrl>https://github.com/emberjs/ember-inspector/issues</bugTrackerUrl>
+    <packageSourceUrl>https://chrome.google.com/webstore/detail/ember-inspector/bmdblncegkenkacieihfhpjfppoconhi</packageSourceUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/emberjs/ember-inspector/blob/master/LICENSE</licenseUrl>
+    <description>
+####PACKAGING NOTE: This installs no software. It installs a registry key for the extension that Chrome will see and then ask you for permission to enable the extension if you are in Chrome or on the next run. Chrome will install the latest version of the extension. The mentioned version is the version at time of packaging, you can ignore it. Chrome will handle updates to the extension.
```

Comment:
Stick to [Informative description guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description)